src/iomusxml.cpp

```diff
@@ -2476,14 +2475,21 @@ void MusicXmlInput::ReadMusicXmlFigures(pugi::xml_node node, Measure *measure, c
         Text *text = new Text();
         text->SetText(UTF8to32(textStr));
         f->AddChild(text);
-        fb->AddChild(f);
+        figures.push_back(f);
+    }
+    if (!figures.empty()) {
```

Comment:
I think it would be a bit clearer here to return if the vector is empty instead of having an `if` block if not empty – since nothing is happening after it anyway

Reply:
sure